### ISSUE TITLE
Can't detect ssid on macOS Mojave 10.14 Public Release

### ISSUE THREAD

**Issue by dedicahyadi** [chunnann/ATH9KFixup#10]
Please fix 1.2.6, it's cannot detect any ssid wifi on 10.14 build 18A391 with my ar956x chipset model.

Thanks in advance

**Comment by MrMangust** [comment]
AR9485 not working too

**Comment by fathurwl** [comment]
Same, ar9565 (dw1707)

**Comment by MrMangust** [comment]
Guys, there are kexts for 9485, 9565 and for 946x. Just put pathced AirPortAtheros40.kext to /System/Library/Extensions/IO80211Family.kext/Contents/PlugIns and delete ATH9KFixup.kext, ATH9KInjector.kext

**Comment by fathurwl** [comment]
I've tried that method, but my wifi card still can't see SSID around me ... how do I fix it ??

I'm on mojave 10.14.1

**Comment by MrMangust** [comment]
Did you delete  ATH9KFixup.kext and ATH9KInjector.kext?

**Comment by fathurwl** [comment]
ya, saya sudah menghapus  kedua file tersebut file tersebut ..

**Comment by fathurwl** [comment]
yes, I have deleted both of these files ..

**Comment by MrMangust** [comment]
And patcher Airport located only in System/Library/Extensions/IO80211Family.kext/Contents/PlugIns? 
No any copies of this kext in System/Library/Extensions?

**Comment by mds88** [comment]
it's not working for ar9565, did what @MrMangust  said

**Comment by dedicahyadi** [comment]
Yes, me too, i've tried to updated. but not working anymore with latest release ath9fixup.
________________________________

**Comment by johnnywolinger** [comment]
The provided kext won't fix the wifi scan in some AR9565, including DELL variation, only got activation, but not possible to scan Wifi and connect to the network, what chunnann or anybody need to got data of this card not scanning network, to implement in kext a possible fix?
